sdk/eventhub/Microsoft.Azure.EventHubs.Processor/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 4.3.0 (Unreleased)
+
+
 ## 4.2.1 (Unreleased)
```

Comment:
If 4.2.1 went unreleased, is there a reason that we'd leave the empty changelog?

Reply:
4.2.1 was a space holder for testing purpose only. I will remove it before the release.